docs/trouble.md

```
@@ -0,0 +1,190 @@
+
+## Speaker Sound
+
+My solution to this exact problem is ChromeCast audios, I have two in a
+group and ChromeCast to the group. One of them is connected to my DAC
+that is then connected to my amplifiers, and another is connected to a
+line in on my computer in a separate room that LedFx is running on.
+
+They\'re perfectly synchronized, but if you need to, you can do a delay
+on any of the ChromeCasts within the group to adjust sync.
+
+### Windows:
```

Markdownlint (0.35.0)

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)